<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="separator"/>
   <addaction name="actAdd"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actSave"/>
   <addaction name="actCancel"/>
   <addaction name="separator"/>
   <addaction name="actFieldList"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
  </action>
  <action name="actAdd">
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="text">
    <string>插入</string>
   </property>
   <property name="toolTip">
    <string>插入</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actCancel">
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消</string>
   </property>
  </action>
  <action name="actFieldList">
   <property name="text">
    <string>字段列表</string>
   </property>
   <property name="toolTip">
    <string>字段列表</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
